frontends/web/src/locales/en/app.json

```diff
@@ -72,7 +72,9 @@
   },
   "aopp": {
     "addressRequest": "{{host}} is requesting a receiving address.",
-    "addressRequestWithLogo": "is requesting a receiving address",
+    "addressRequestWithLogo": "is requesting a receiving address.",
+    "addressRequestWithLogoAndXPub": "is requesting a receiving address and public key (xPub).",
+    "addressRequestWithXPub": "{{host}} is requesting a receiving address and public key (xPub).",
```

Comment:
Please lowercase "xpub" so it is consistent with existing messages / in-app guide etc.